src/features/components/molecules/instruments-table/instrumental-table.module.scss

```diff
@@ -0,0 +1,27 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+.instrumental_wrapper {
+    border: 1px solid $color-grey-6;
```

Comment:
Can you check for the variable name used for $color-grey-6 and use that

Reply:
i coudnt find a variable name for this 
src/features/styles/theme/buttons.scss
saw similar usage in this file too